qualtran/bloqs/arithmetic.py

```diff
@@ -194,9 +203,7 @@ class GreaterThan(Bloq):
      - result: A single bit output register to store the result of A > B.
 
     References:
-        [Improved techniques for preparing eigenstates of fermionic
-        Hamiltonians](https://www.nature.com/articles/s41534-018-0071-5#additional-information),
-        Comparison Oracle from SI: Appendix 2B (pg 3)
+        See cirq_ft comparitors.
```

Comment:
probably don't include this under the "References" section. You can probably just put it in the body of the docstring. And it would be good to explain what exactly I should be seeing about the cirq_ft comparators